application/event_processor/reporting.py

```diff
@@ -89,17 +92,27 @@ def log_invalid_nhsuk_pharmacy_postcode(nhs_entity: NHSEntity, dos_service: DoSS
             "dos_service": dos_service.uid,
         },
     )
-
-
-def log_invalid_open_times(nhs_entity: NHSEntity, matching_services: List[DoSService]) -> None:
+    metrics.set_namespace("UEC-DOS-INT")
+    metrics.set_property("level", "WARNING")
+    metrics.set_property("message", error_msg)
+    metrics.set_property("correlation_id", logger.get_correlation_id())
+    metrics.set_property("ods_code", nhs_entity.odscode)
+    metrics.set_dimensions({"ENV": environ["ENV"]})
+    metrics.put_metric("InvalidPostcode", 1, "Count")
+    metrics.set_property("level", "INFO")
```

Comment:
Why setting the level as both warning and info, fairly sure this could lead to everything being info as will be the value when the metrics are flushed

Reply:
Let me have a look